llvm/lib/Target/ARM/AsmParser/ARMAsmParser.cpp

```diff
@@ -6173,7 +6187,7 @@ ParseStatus ARMAsmParser::parseFPImm(OperandVector &Operands) {
 
   Parser.Lex(); // Eat '#' or '$'.
 
-  // Handle negation, as that still comes through as a separate token.
+  // Handle negation, as that still comes through as a  token.
```

Comment:
Oops. I think you need another commit, see other instances below ...

Reply:
Yeah just added them all back, and ran the formatter properly. Sorry about that!